### PR TITLE
docs update

### PR DESCRIPTION
**Pull request by prettyirrelevant**

### CODE REVIEW

File: docs/installation_guide.rst [hunk 1]
```diff
@@ -472,7 +472,7 @@ then do::
     Make sure that pysqlcipher3 is properly installed. If ``$ pip3 freeze | grep pysqlcipher3`` returns nothing for you then it was not installed. Try to manually install only that dependency with the verbose option to see where it fails. ``$ pip3 install pysqlcipher3 -v``. If it fails at the stage of finding the library for ``-lsqlcipher`` or ``sqlcipher/sqlite3.h file not found`` then ``brew install sqlciper`` did not place the installed lib directory to the ``LIBRARY_PATH`` and you will have to do it manually.
     For example, if ``sqlcipher`` was installed using Homebrew, the library path resembles this ``/opt/homebrew/Cellar/sqlcipher/<version>/lib`` on Mac ARM based chips and ``/usr/local/Cellar/sqlcipher/<version>/lib`` on Mac Intel based chips by default.
     Afterwards, install ``pysqlcipher3`` using the command below:
-    ``$ LIBRARY_PATH=/opt/homebrew/Cellar/sqlcipher/4.5.1/lib pip3 install pysqlcipher3``.
+    ``$ LIBRARY_PATH=/opt/homebrew/Cellar/sqlcipher/4.5.1/lib C_INCLUDE_PATH=/opt/homebrew/Cellar/sqlcipher/4.5.1/include pip3 install pysqlcipher3``.
```
Comment (LefterisJP) [inline docs/installation_guide.rst:475]: what is the `C_INCLUDE_PATH` needed for? We have not had the need to use it (afaik) in our macos CI building. @kelsos ?

File: docs/usage_guide.rst [hunk 2]
```diff
@@ -70,7 +70,7 @@ If you already have an account just write the name and the password at the sign
 Set up rotki Premium
 ======================
 
-If you decide to purchase `rotki Premium <https://rotki.com/products/>`_ at a later time, you can set it up via API Keys -> rotki Premium.
+If you decide to purchase `rotki Premium <https://rotki.com/products/>`_ at a later time, you can set it up via :menuselection:`API Keys --> rotki Premium`.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:73]: what does the menuselection do here?

File: docs/usage_guide.rst [hunk 3]
```diff
@@ -99,7 +99,7 @@ Bear in mind that in case of using multiple accounts/devices with the data sync
    :alt: Replace local database with remote backup
    :align: center
 
-You can manually move the global DB that contains the assets from one system to the other too. Find the :ref:`rotki_data_directory` in the source system. Assuming it's linux it will be ``~/.local/share/rotki/data``. The global db is then ``~/.local/share/rotki/data/global_data/global.db``. Manually move it to the equivalent location in the new system.
+You can manually move the global DB that contains the assets from one system to the other too. Find the :ref:`rotki_data_directory` in the source system. Assuming it's linux it will be :file:`~/.local/share/rotki/data`. The global db is then :file:`~/.local/share/rotki/data/global_data/global.db`. Manually move it to the equivalent location in the new system.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:102]: or this file thing?

**Review by LefterisJP — COMMENTED** [review]
Hey so for section 6 "Customization of the list of supported assets", shouldn't you also mention the new fact that you can ignore an asset by clicking on the new checkbox on the asset page?

File: docs/usage_guide.rst (hunk 2 above)
Comment (prettyirrelevant) [reply docs/usage_guide.rst:73]: It turns it into a breadcrumb.

## Example
From this:
Screenshot 2022-05-27 at 09 23 16

To this:
Screenshot 2022-05-27 at 09 21 53

**Review by nebolax — COMMENTED** [review]
`docs/images/rotki_add_edit_token.png` seems to be not 1664 x 1162. Was it made on purpose?

`docs/images/rotki_premium_set_sync_data.png` is not 1664 x 1162 too. And as I see the content on this screenshot is exactly the same as on it's previous version, only resolution has changed.

Also in my opinion 3 screenshots for rotki premium is too much as we could use one where fields for API credentials, database sync switch and disable premium button will be shown. What do you think @prettyirrelevant @LefterisJP ?

File: docs/usage_guide.rst (hunk 3 above)
Comment (prettyirrelevant) [reply docs/usage_guide.rst:102]: The addition does not make a visual representation difference.
But it was from the [https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html?highlight=kbd#role-file](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html?highlight=kbd#role-file).

File: docs/installation_guide.rst (hunk 1 above)
Comment (prettyirrelevant) [reply docs/installation_guide.rst:475]: Not adding the `C_INCLUDE_PATH` did not install `pysqlcipher3` for me. 
>In file included from src/python3/connection.c:27:
    src/python3/connection.h:33:10: fatal error: 'sqlcipher/sqlite3.h' file not found
    #include "sqlcipher/sqlite3.h"
             ^~~~~~~~~~~~~~~~~~~~~
    1 error generated.
    error: command '/usr/bin/clang' failed with exit code 1
    Running setup.py install for pysqlcipher3 ... done
Successfully installed pysqlcipher3
WARNING: You are using pip version 21.2.4; however, version 22.1.1 is available.
You should consider upgrading via the '/Users/Isaac/Desktop/rotki/.venv/bin/python -m pip install --upgrade pip' command.
❯ python -c "import pysqlcipher3"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'pysqlcipher3'

**Comment by prettyirrelevant** [comment]
> Hey so for section 6 "Customization of the list of supported assets", shouldn't you also mention the new fact that you can ignore an asset by clicking on the new checkbox on the asset page?

There's no checkbox to ignore assets on the `Manage Assets` page.

**Comment by LefterisJP** [comment]
> > Hey so for section 6 "Customization of the list of supported assets", shouldn't you also mention the new fact that you can ignore an asset by clicking on the new checkbox on the asset page?
> 
> There's no checkbox to ignore assets on the `Manage Assets` page.

There is on the asset details of each asset, at which you can go from the manage assets page

File: docs/usage_guide.rst (hunk 2 above)
Comment (LefterisJP) [reply docs/usage_guide.rst:73]: ah very cool

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**